examples/src/document_loaders/unstructured.ts

```diff
@@ -2,7 +2,7 @@ import { UnstructuredLoader } from "langchain/document_loaders";
 
 export const run = async () => {
   const loader = new UnstructuredLoader(
-    "http://localhost:8000/general/v0.0.4/general",
+    "http://localhost:8000/general/v0/general",
```

Comment:
Why this change @jasondotparse ?

Reply:
I think the url was wrong. I tried it myself and I could only get Unstructured to work with my local docker instance after removing the ".0.4"

Reply:
I probably should have made that a separate PR though. I'll remove this change since it's not relevant to the rest of the S3 feature.